video/out/opengl/context_angle.c

```diff
@@ -646,6 +646,7 @@ static int angle_control(struct ra_ctx *ctx, int *events, int request, void *arg
 const struct ra_ctx_fns ra_ctx_angle = {
     .type           = "opengl",
     .name           = "angle",
+    .description    = "Direct3D 11 through OpenGL ES (ANGLE)",
```

Comment:
Isn't it the other way. OpenGL through ANGLE? And I don't think it necessary is d3d11. I would call it `OpenGL/ANGLE`

Reply:
👍